ibis/tests/all/test_numeric.py

```diff
@@ -314,6 +316,7 @@ def test_mod(backend, alltypes, df):
     backend.assert_series_equal(result, expected, check_dtype=False)
 
 
+@pytest.mark.skip_backends((MSSQL,))
```

Comment:
any reason to use `skip_backends` here instead of `xfail_backends`?

Reply:
no reason at all. changed to xfail_backends